java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1171,6 +1171,48 @@ public void zscore(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zdiff(BaseClient client) {
+        String key1 = "{testKey:}" + UUID.randomUUID();
```

Comment:
This could be `{testKey}` or `{testKey}:` or `{testKey}-` or anything
Just keep common part in `{}`